### ISSUE TITLE
Android projects fail when using multiple API libraries due to misplaced MANIFEST file

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/google-api-java-client#761]
```
Version of google-api-java-client (e.g. 1.5.0-beta)?
google-api-java-client-1.14.1-beta

Java environment (e.g. Java 6, Android 2.3, App Engine)?
JDK 1.6.0.23, Android 4.2

Describe the problem.
When including google-api-services-drive-v2-rev63-1.14.1-beta.jar and 
google-api-services-youtube-v3-rev40-1.14.1-beta.jar in libs/ folder of Android 
APK project, the project failed to build because of the error in Eclipse console
Error generating final archive: Found duplicate file for APK: MANIFEST.MF
Origin 1: \libs\google-api-services-drive-v2-rev63-1.14.1-beta.jar
Origin 2: \libs\google-api-services-youtube-v3-rev40-1.14.1-beta.jar


How would you expect it to be fixed?
google-api-services-youtube-v3-rev40-1.14.1-beta.jar and 
google-api-services-drive-v2-rev63-1.14.1-beta.jar should have MANIFEST.MF 
inside META-INF/ (not outside as currently). This issue might also affect to 
other API jars also.
```

Original issue reported on code.google.com by `hathanht...@gmail.com` on 31 Mar 2013 at 5:26

**Comment by GoogleCodeExporter** [comment]
```
+1. i solved temporarily by using 1.13-2 versions. Seems like all 1.14-1 
service jars have MANIFEST.MF
```

Original comment by `uluk...@google.com` on 10 Apr 2013 at 4:03

**Comment by GoogleCodeExporter** [comment]
Original comment by `ngmic...@google.com` on 10 Apr 2013 at 5:09
- Changed state: **Accepted**
- Added labels: **Priority-High**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
Original comment by `ngmic...@google.com` on 10 Apr 2013 at 5:26
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
The fix will be found in 1.14.2-beta service jars, which will be released later 
today.  The core library will remain at 1.14.1-beta. 

Thanks for the bug report.
```

Original comment by `ngmic...@google.com` on 22 Apr 2013 at 4:17
- Changed title: **Android projects fail when using multiple API libraries due to misplaced MANIFEST file**
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
well done trai :)
```

Original comment by `qp...@marketlive.com` on 23 Apr 2013 at 2:11
